xcm/pallet-xcm/src/lib.rs

```diff
@@ -52,8 +52,8 @@ use frame_system::pallet_prelude::*;
 pub use pallet::*;
 use xcm_executor::{
 	traits::{
-		CheckSuspension, ClaimAssets, DropAssets, FinishedQuery, MatchesFungible, OnResponse, QueryResponseStatus,
-		VersionChangeNotifier, WeightBounds, XcmQueryHandler,
+		CheckSuspension, ClaimAssets, DropAssets, MatchesFungible, OnResponse, QueryHandler,
+		QueryStatus as OuterQueryStatus, VersionChangeNotifier, WeightBounds,
```

Comment:
Why did you alias it to OuterQueryStatus?

Reply:
there is already a QueryStatus type inside the pallet. and I've seen in Frame we use Outer suffix (not exactly same case).